src/utils/mapUtils.tsx

```diff
@@ -100,13 +102,15 @@ export const highlightFeature = (e: LeafletMouseEvent) => {
     weight: 5,
     color: '#666',
     dashArray: '',
-    fillOpacity: 0.7,
-    zIndex: 200
+    fillOpacity: 0,
+    //zIndex: 200
   });
-  layer.bringToFront();
+  layer.bringToBack();
 
 }
 
+export const resetHighlightVectorFeature = (e: LeafletMouseEvent) => e.target.resetFeatureStyle(e.layer.properties.CODE);
+
 export const resetHighlight = (e: LeafletMouseEvent) => {
   const layer = e.target;
 

```

Comment:
empty line

Reply:
bro as a code artisan myself, i am a strong believer of letting the code stretch out and breath

Reply:
agreed lol